SitecoreDeployer/buildtask/functions-module-v1.psm1

```diff
@@ -195,15 +195,15 @@ function CollectWebAppOutboundIpAddresses{
 
     if (!$resourcePresenceChecked) {
         #get all resrouces in current resource group, and check if resource is present there
-        $webAppResource = (Find-AzureRmResource -ResourceGroupNameContains $resourceGroupName).where({$_.Name -eq "$webAppName" -And $_.ResourceGroupName -eq "$resourceGroupName"})
+        $webAppResource = (Get-AzureRmResource -ResourceGroupName *$resourceGroupName*).where({$_.Name -eq "$webAppName" -And $_.ResourceGroupName -eq "$resourceGroupName"})

```

Comment:
Please, remove asteriks here - we need resources from specific resource group, not matching one (in Find-AzureRmResource there was not possibility to match exact rg name)